server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/BlobTables.java

```diff
@@ -48,4 +48,11 @@
         String CHUNK_NUMBER = "chunkNumber";
         String DATA = "data";
     }
+
+    interface DumbBlobCache {
+        String TABLE_NAME = "blobCache";
```

Comment:
should we start using snake case for new cassandra tables?

Reply:
Name this table `blob_cache` please